docs/development.md

```diff
@@ -19,7 +19,7 @@ Please follow the instructions from the [local tutorial](local_tutorial.md#setup
 
 ### Install the necessary dependencies
 
-Install the Zookeeper & Etcd Operators, which this operator depends on by default.
```

Comment:
@HoustonPutman
I suppose you didn't want to remove the word "Operators".

Reply:
Oops, good call there.